manifest.md

```diff
@@ -140,9 +140,8 @@ Unlike the [Manifest List](#manifest-list), which contains information about a s
 - **`layers`** *array*
 
     Each item in the array MUST be a [descriptor](descriptor.md).
-    The array MUST have the base image at index 0.
-    Subsequent layers MUST then follow in the order in which they are to be layered on top of each other.
-    The algorithm to create the final unpacked filesystem layout MUST be to first unpack the layer at index 0, then index 1, and so on.
```

Comment:
@wking Please respond inline on the review.

> Do
you have a particular feature in the lines I'm removing that you feel
it doesn't cover?

You've changed the emphasis and meaning of the clause.

Before, the emphasis was on ordering, where now it is focusing on the nature of the base layer.